app/javascript/components/enrollment/Enrollment.js

```diff
@@ -43,6 +43,11 @@ class Enrollment extends React.Component {
   }
 
   setEnrollmentState = debounce(enrollmentState => {
+    // save jurisdiction_id as a number if it is in this enrollmentState
```

Comment:
This shouldn't be necessary to do here. What should happen is we should parse the jurisdiction_id correctly as an integer when we actually change it in `Exposure.js` here: https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/components/enrollment/steps/Exposure.js#L49. That line should safely check and convert that from a string to an integer and then the change here won't be necessary.

Reply:
Good catch